libs/broker-db/src/converter.test.ts

```diff
@@ -0,0 +1,59 @@
+import { init, filterByEmulator, getFirestore, Timestamp } from '@thecointech/firestore';
+import { describe } from '@thecointech/jestutils';
+import { getUserData, setUserVerified } from 'user';
+import { DateTime } from 'luxon';
+import { userDataConverter } from 'user.types';
+import { getActionFromInitial } from 'transaction';
+import Decimal from 'decimal.js-light';
+import { PurchaseType } from 'transaction/types';
+
+describe('Live DB fetching', () => {
+
+  beforeAll(() => {
+    init();
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  })
+
+  // Test Address.  Uses Timestamp to give uniqueness because tests
+  // do not clean up after themselves.
```

Comment:
we allow these tests to connect with the emulator (if its running) because I wanted to be able to see what the results actually looked like. The emulator doesn't clean up after itself